### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by iawale**

### CODE REVIEW

**Comment by iawale** [comment]
image

**Comment by rojan96** [comment]
Minor conflict but looks good otherwise!

File: App.tsx [hunk 1]
```diff
@@ -13,6 +13,43 @@ export default function App() {
   );
 }
 
+const dummyData = [
```
Comment (rojan96) [inline App.tsx:16]: This is redundant, please remove. or else.

File: src/service/UserService.ts [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
-      img: {uri: 'https://scontent-atl3-1.xx.fbcdn.net/v/t31.18172-8/26685663_10204597532775805_3451693112975209419_o.jpg?_nc_cat=110&ccb=1-7&_nc_sid=a4a2d7&_nc_ohc=EKLktQvWoZAAX9CSZml&_nc_ht=scontent-atl3-1.xx&oh=00_AfB7jflOaBf01tA_qOqaPoZJsHZeWnqdHNmBsYBVGldERw&oe=6424F427'}
+  {
+    name: "Chirag Awale",
+    img: {
+      uri: "https://scontent-atl3-1.xx.fbcdn.net/v/t31.18172-8/26685663_10204597532775805_3451693112975209419_o.jpg?_nc_cat=110&ccb=1-7&_nc_sid=a4a2d7&_nc_ohc=EKLktQvWoZAAX9CSZml&_nc_ht=scontent-atl3-1.xx&oh=00_AfB7jflOaBf01tA_qOqaPoZJsHZeWnqdHNmBsYBVGldERw&oe=6424F427",
     },
-    {
-      name: 'Rojan Maharjan',
-      img: {uri: 'https://scontent-atl3-1.xx.fbcdn.net/v/t39.30808-6/296214048_5822958491066280_5883667737777082916_n.jpg?_nc_cat=105&ccb=1-7&_nc_sid=730e14&_nc_ohc=U-nO5ioE1gsAX-GEOo1&tn=dlJHKpBOJpfVyFPh&_nc_ht=scontent-atl3-1.xx&oh=00_AfDtnQs5yyNPtfiElBEZ6Ul7zTeyfamA-aQRsUvN5o6eJQ&oe=6402EAA6'}
+    bio: "Hey! I am looking for a roomate in the Atlanta Area.",
```
Comment (rojan96) [inline src/service/UserService.ts:7]: Not sure why everyone has the same bio, but this time I'll approve.

**Review by rojan96 — APPROVED** [review]

**Pull request merged**